cmd/config/env_var_reader.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/flyteorg/flyte/flyteidl/clients/go/admin"
+	"github.com/flyteorg/flyte/flytestdlib/config"
+)
+
+const flyteAdminEndpoint = "FLYTE_ADMIN_ENDPOINT"
+
+type FuncType func() error
+
+var funcMap = map[string]FuncType{flyteAdminEndpoint: updateAdminEndpoint}
+
+func UpdateConfigWithEnvVar() error {
+	for envVar, f := range funcMap {
+		if os.Getenv(envVar) != "" {
+			if err := f(); err != nil {
+				return fmt.Errorf("error update config with env var: %v", err)
+			}
+		}
+	}
+	return nil
+}
+
+func updateAdminEndpoint() error {
+	ctx := context.Background()
```

Comment:
If we have multiple env var in the future, will initializing ctx every single time be a problem?